src/components/Home/Cart.tsx

```
@@ -346,12 +366,16 @@ const Cart = ({ actionData, orderData, states, crewData, unpaidReports, calculat
                   <Select labelId="demo-select-small-label" id="demo-select-small" label="Payment" value={paymentMethod} onChange={handlePaymentMethodChange} error>
-                    <MenuItem value="Cash">Cash</MenuItem>
+                    {methods.map((method: any) => {
+                      return (<MenuItem value={method.name}>{method.name}</MenuItem>)
+                    })}
                     <MenuItem value="Gift Card">Gift Card</MenuItem>
                   </Select>
                 ) : (
                   <Select labelId="demo-select-small-label" id="demo-select-small" label="Payment" value={paymentMethod} onChange={handlePaymentMethodChange}>
-                    <MenuItem value="Cash">Cash</MenuItem>
+                    {methods.map((method: any) => {
+                      return (<MenuItem value={method.name}>{method.name}</MenuItem>)
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)